src/js/markup-injectors/update-tasklist-dom-injection.js

```diff
@@ -0,0 +1,18 @@
+import TaskManager from '../user-controller/task-manager-class.js';
+import createTaskTemplate from '../markup-templates/create-task-template.js';
+
+const updateTaskListOnDOM = (updateWhen) => {
+  const taskListHolder = document.querySelector('[data-task-list-holder]');
+  const taskList = new TaskManager().getTaskList();
+  if (updateWhen === 'lastTask') {
+    taskListHolder.innerHTML += createTaskTemplate(
+      taskList[taskList.length - 1]
+    );
+  } else if (updateWhen === 'fullList') {
+    taskList.forEach((task) => {
+      taskListHolder.innerHTML += createTaskTemplate(task);
+    });
+  }
+};
```

Comment:
For this module, I will suggest the use of two different functions, as I think it would be more clear and easy to manage from one side the creation of a new task, and from the other to display in the HTML.

Reply:
Hi @HFG43 Hernan!, yes, you are right, I should split this function into two different ones. I will make the respective changes right away 🎯